src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -40,7 +50,7 @@ public GitHubAccessor(Config config)
 
             if (!string.IsNullOrEmpty(config.GithubToken))
             {
-                _httpClient = new HttpClient();
+                _httpClient = s_sharedHttpClient;
```

Comment:
This http client cannot be shared:
```suggestion
                _httpClient = new(new HttpClientHandler { CheckCertificateRevocationList = true });
```